GuildedRoseLLCSpec/ItemCollectionViewDataSourceSpec.swift

```diff
@@ -34,8 +34,16 @@ class ItemCollectionViewDataSourceSpec: QuickSpec {
             }
             
             it("has six items") {
```

Comment:
Question, no changes required: Why are your tests covering zero items and then six items? It seems kind of weird that you are testing 0 and 6, but nothing in between?

Reply:
Do we need additional item numbers aside from zero, one (line 27), and many (6 items)?